presto-manager-agent/src/main/java/com/teradata/prestomanager/agent/LogsHandler.java

```diff
@@ -44,38 +45,52 @@
 /**
  * Utility class managing retrieval and deletion of Presto logs
  */
-public final class LogsHandler
+public class LogsHandler
 {
     private static final Logger LOG = Logger.get(LogsHandler.class);
 
     public static final String DEFAULT_LOG_LEVEL = "ALL";
 
-    // TODO: Make most of these 'private static final' values configurable
-    private static final Path LOG_DIRECTORY = Paths.get("var/log/presto");
 
     private static final DateTimeFormatter DATE_FORMAT = new DateTimeFormatterBuilder()
             .parseCaseInsensitive().parseStrict()
             .append(DateTimeFormatter.ISO_LOCAL_DATE_TIME)
             .appendPattern("Z")
             .toFormatter().withChronology(IsoChronology.INSTANCE);
 
-    // TODO: When these are configurable, check that the group names are present
     private static final String DATE_GROUP = "date";
     private static final String LEVEL_GROUP = "level";
```

Comment:
Added. I also added detailed descriptions to the configuration files themselves.